geomstats/geometry/special_euclidean.py

```diff
@@ -36,13 +38,46 @@
                         - 1. / 362880.]
 
 
+def embed(rotation, translation, output_shape, constant=1.):
+    r"""Embed rotation, translation couples into n+1 square matrices.
+
+    Construct a block matrix of size :math: `n + 1 \times n + 1` of the form
+    ((R, t), (0, c)).
+
+    Parameters
+    ----------
+    rotation : array-like, shape=[..., n, n]
+        Square Matrix.
+    translation : array-like, shape=[..., n]
+        Vector.
+    output_shape : tuple of int
+        Desired output shape. This is need for vectorization.
+    constant : float
+        Constant to use at the last line and column of the square matrix.
+        Optional, default: 1
+
+    Returns
+    -------
+    mat: array-like, shape=[..., n + 1, n + 1]
+        Squre Matrix of size n + 1. It can represent an element of the
```

Comment:
Squre -> Square

Reply:
Not sure I understand this docstring? special euclidean group of the Lie algebra?

Reply:
Nit: it can represent an element of SE(3) or its Lie aglebra